nanoFramework.System.Net.Http/Http/System.Net.HttpWebRequest.cs

```diff
@@ -1476,14 +1507,14 @@ private InputNetworkStreamWrapper EstablishConnection(Uri proxyServer, Uri targe
                     retStream.m_rmAddrAndPort = m_originalUrl.Host + ":" + m_originalUrl.Port;
                 }
 
-                lock (m_ConnectedStreams)
-                {
-                    m_ConnectedStreams.Add(retStream);
+                if (m_keepAlive) {              // skigrinder - check keepAlive before creating persistent connection - persistent connections are not currently supported
```

Comment:
Nice catch!
Please move the open bracket to the next line. That's the code style that the project is following.

Reply:
Thanks.   Sorry.